src/Analysis/Engine/Impl/Interpreter/Ast/NameLookupContext.cs

```diff
@@ -228,28 +224,40 @@ sealed class NameLookupContext {
         }
 
         private IMember GetValueFromMember(MemberExpression expr, LookupOptions options) {
-            if (expr == null || expr.Target == null || string.IsNullOrEmpty(expr.Name)) {
+            if (expr?.Target == null || string.IsNullOrEmpty(expr?.Name)) {
                 return null;
             }
 
             var e = GetValueFromExpression(expr.Target);
-            if (e is IMemberContainer mc) {
-                var m = mc.GetMember(Context, expr.Name);
-                if (m != null) {
-                    return m;
-                }
+            IMember value = null;
+            switch (e) {
+                case IMemberContainer mc:
+                    value = mc.GetMember(Context, expr.Name);
+                    break;
+                case IPythonMultipleMembers mm:
+                    value = mm.Members.OfType<IMemberContainer>()
+                        .Select(x => x.GetMember(Context, expr.Name))
+                        .ExcludeDefault()
+                        .FirstOrDefault();
+                    break;
+                default:
+                    value = GetValueFromPropertyOrFunction(e, expr);
+                    break;
+            }
+            if(value is IBuiltinProperty2 p) {
+                value = GetPropertyReturnType(p, expr);
```

Comment:
`GetPropertyReturnType` never returns null, so `value == null` can be tested earlier.